### DISCUSSION TITLE
Is there a way to not display the fames menu ?

### DISCUSSION THREAD

**Discussion by NedNet29** [senshu/Sozi#684] [Q&A] [answered]
Hi all,

I use Sozi to create simple zoomable images in web pages, so all I do is open my SVGs with Sozi, uncheck whatever interactions I don't want (typically backspace keyboard behavior and rotations), and I don't even create a single view/frame ("vue" in french UI).
I then use the generated html page to be embedded in another page.

What I wish is not seeing the frame menu that in my case contains just a white "1" on black bckaground, and displays a (legitimately) empty menu when I click on it.

Is there a way to do this with Sozi, and if not should I suggest/ask for it in the issues ?

**Comment by ejvindh** [comment, marked as answer]
I don't think this is currently part of the Sozi-tool. And I don't know if aumouvantsillage is willing to implement it.

As a workaround (and, admittedly not one of the most elegant) you can open the html-file and set the sozi-frame-list-class to invisible in the css:
visibility: hidden;

Also find this sequence of code:
function(){r.playing?document.title=t.title:document.title=t.title+" (Paused)"}
and replace with:
function(){}

This gives you what you want. I'm sure it can be done in a more elegant way, but this would demand a better understanding of the underlying code on my part :)

**Reply by NedNet29** [reply]
Thank's a lot !

**Comment by aumouvantsillage** [comment]
Based on the description of your problem, I understand that you want to hide the current frame number when the "presentation" is played, but I'm not sure this is exactly what you meant.

You can tell Sozi to hide the current frame number for a given frame by clicking the "#" button at the top of the properties panel in the presentation editor.
